frontend/src/entity-models/access-token.ts

```diff
@@ -0,0 +1,3 @@
+export interface AccessToken {
+    accessToken: string;
+}
```

Comment:
eof fix pls

Reply:
Idk why new lines are not shown in code. I have 'em in my VScode, but they are not shown here